services/m3dbnode/server/serve.go

```diff
@@ -38,9 +38,9 @@ import (
 	"github.com/m3db/m3db/ts"
 )
 
-const defaultNamespaceName = "default"
+const defaultNamespaceName = "metrics"
```

Comment:
Might be better to use something more generic than "metrics" since people can use m3db for other purposes